certification/pyxis/submit.go

```diff
@@ -57,6 +57,15 @@ func (p *pyxisEngine) SubmitResults(ctx context.Context, certProject *CertProjec
 		}
 	}
 
+	for _, artifact := range artifacts {
+		artifact.ImageID = certImage.ID
+		_, err = p.createArtifacts(ctx, &artifact)
```

Comment:
If we aren't using the return value, collapse the if statement to this line.

Reply:
fixed